garageScript/c0d3-app
# fix: Button component background color

Closes #2206 

Merging this PR will fix the `Button` component background color when the element is in focus or active state. 

Currently, it looks like this on focus or active states:

<img width="251" alt="image" src="https://user-images.githubusercontent.com/35906419/189100631-5eb4cc07-0d1b-4477-8689-10c22c91ffe4.png">

After merging this PR:

<img width="250" alt="image" src="https://user-images.githubusercontent.com/35906419/189100700-3a9d0a71-0445-4c91-aae2-652484deecbc.png">

**To test it:**
1. Click the button (waiting for the storybook link to be generated)
2. Move the cursor away from it
3. See how the background isn't filled with the button color
